templates/site/datacatalog.html

```diff
@@ -28,7 +28,7 @@
                 })
             }
 
-            function setNodeLink(name, data, node_id_suffix) {
+            function setNodeLink(nodeName, data, node_id_suffix) {
```

Comment:
This isn't really the node's name anymore though. It's a unique id that may or may not have anything to do with the name.

Calling this variable `nodeName` is going to be confusing. I'd change it to `nodeID` (and elsewhere)

Reply:
Changed to nodeID